src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -961,11 +961,11 @@ namespace winrt::Microsoft::Terminal::Control::implementation
 
         auto bufferHeight = _core.BufferHeight();
 
-        ScrollBar().Maximum(bufferHeight - bufferHeight);
+        ScrollBar().Maximum(0);
         ScrollBar().Minimum(0);
         ScrollBar().Value(0);
         ScrollBar().ViewportSize(bufferHeight);
-        ScrollBar().LargeChange(std::max(bufferHeight - 1, 0)); // scroll one "screenful" at a time when the scroll bar is clicked
+        ScrollBar().LargeChange(bufferHeight); // scroll one "screenful" at a time when the scroll bar is clicked

```

Comment:
there must have been a reason it was bufferHeight-1, right?

Reply:
I _think_ it could be because the original author compared the behavior against `less`, where PgUp/Down moves by 29 lines in a 30 line viewport. But that's because the 30th line is the ":" command prompt. I tested this and setting this to `bufferHeight` does do what you'd think it does (= page by 30 rows / the viewport height).